actions/helm-lint/action.yaml

```diff
@@ -12,11 +12,6 @@ outputs:
 runs:
   using: "composite"
   steps:
-    - name: Checkout repository
```

Comment:
There are a lot of other checkouts made in other actions, are we going by the best practice and removing all of them?

Reply:
Removed

Reply:
There is still a `checkout` in `update-artifacts-file` action.
In addition, there is a `git checkout` command in `commit_changes.sh`, I think it isn't needed anyway because the checkout should already be for the wanted branch.

Reply:
Done

Reply:
There is still the `checkout` step in the action itself.